README.md

```diff
@@ -342,6 +435,17 @@ Publishes a new helm chart package (`.tgz`) to a helm chart repository
           token: ${{ secrets.BOT_GITHUB_TOKEN}}
 ```
 
+### release-descriptor-loader
```

Comment:
I should have commented on this earlier, sorry, but does it make sense to rename it to "load-release-descriptor"? Other actions due use a verb, and the "release" word can be misleading here.

Reply:
It makes sense. Thanks. Fixed.